aws/environments/dev/terragrunt.hcl

```diff
@@ -5,7 +5,9 @@ terraform {
     required_var_files = [
       "${get_parent_terragrunt_dir()}/${path_relative_to_include()}/${find_in_parent_folders("region.tfvars")}",
       "${get_parent_terragrunt_dir()}/${path_relative_to_include()}/${find_in_parent_folders("environment.tfvars")}",
-      "${get_parent_terragrunt_dir()}/${path_relative_to_include()}/${find_in_parent_folders("account.tfvars")}"
+      "${get_parent_terragrunt_dir()}/${path_relative_to_include()}/${find_in_parent_folders("account.tfvars")}",
+      "${get_parent_terragrunt_dir()}/${path_relative_to_include()}/${find_in_parent_folders("p-rep-configure.tfvars")}",
```

Comment:
This is going to break other people's configurations as that file is not found in the parent of each module.